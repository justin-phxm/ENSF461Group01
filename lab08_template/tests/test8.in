793
283
735
755
637
266
826
175
107
651
466
949
754
60
52
958
3
263
592
511
316
784
45
139
398
545
976
394
38
403
689
182
