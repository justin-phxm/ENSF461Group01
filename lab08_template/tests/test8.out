793
1076
1811
2566
3203
3469
4295
4470
4577
5228
5694
6643
7397
7457
7509
8467
8470
8733
9325
9836
10152
10936
10981
11120
11518
12063
13039
13433
13471
13874
14563
14745
